frontend/src/pages/HomePage.js

```diff
@@ -1,22 +1,166 @@
-import React, { Component } from 'react';
-import QuestionPage from './QuestionPage';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import React from 'react';
+import { Grid, Paper, Typography, ListItem, List } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles/';
+import theme from '../globalTheme';
+import IconButton from '../components/buttons/IconButton';
+import { ReactComponent as ExamIcon } from '../assets/examIcon.svg';
+import { ReactComponent as ProfileIcon } from '../assets/profileIcon.svg';
+import { ReactComponent as LeaderboardIcon } from '../assets/leaderboardIcon.svg';
+import { ReactComponent as AnswersIcon } from '../assets/answersIcon.svg';
 
-export default class HomePage extends Component {
-	constructor(props) {
-		super(props);
-	}
+const useStyles = makeStyles(() => ({
+	paper: {
+		width: '88%',
+		borderRadius: 40,
+		border: '3px solid #D9D9D9',
+		boxShadow: '-6px 4px 6px rgba(0, 0, 0, 0.25)',
+		margin: '0 auto',
+	},
+	container: {
+		width: '100%',
+		height: '100%',
+		padding: '3rem',
+	},
+	icon: {
+		fontSize: 100,
+	},
+	bold: {
+		fontWeight: 600,
+	},
+	futureText: {
+		fontWeight: 500,
+		margin: '0px 10px',
+	},
+}));
 
-	render() {
-		return (
-			<Router>
-				<Switch>
-					<Route exact path='/test'>
-						<p>This is the home page</p>
-					</Route>
-					<Route path='/question/:id' component={QuestionPage} />
-				</Switch>
-			</Router>
-		);
-	}
-}
+export const HomePage = () => {
+	const classes = useStyles();
+
+	return (
+		<Paper theme={theme} className={classes.paper}>
+			<Grid
+				container
+				direction='column'
+				justifyContent='flex-start'
+				alignItems='flex-start'
+				className={classes.container}
+				spacing={4}
+			>
+				<Grid item xs={1}>
+					<Typography className={classes.bold} variant='h4'>
+						{' '}
+						O que fazer ...
+					</Typography>
+				</Grid>
+
+				{/* Home Buttons */}
+				<Grid style={{ margin: '20px 0px' }} item xs={5}>
+					<Grid
+						spacing={5}
+						container
+						direction='row'
+						justifyContent='flex-start'
+						alignItems='center'
+					>
+						<Grid item xs={3}>
+							<IconButton
+								direction='column'
+								fontSize={25}
+								text='Resolver exames'
+								iconFirst={false}
+								alignItems='flex-start'
+								height='23vh'
+								width='12vw'
+								href='/exames'
+							>
+								<ExamIcon className={classes.icon} />
+							</IconButton>
+						</Grid>
+						<Grid item xs={3}>
+							<IconButton
+								direction='column'
+								fontSize={25}
+								text='Ver o teu perfil'
+								iconFirst={false}
+								alignItems='flex-start'
+								height='23vh'
+								width='12vw'
+								href='/perfil'
+							>
```

Comment:
![image](https://user-images.githubusercontent.com/39437433/169698270-812e96c3-1e7c-45da-9cac-aa4de312f82a.png)
This icon is misaligned with the rest

Reply:
Since it's a grid the distance will depend on the overflow of the text. But I will tr to find a solution

Reply:
@JeronimoMendes Check It out Now
![image](https://user-images.githubusercontent.com/80652363/169834803-a9dc9ad2-883f-4e8c-9fc5-17ea19cfa6e0.png)